scripts/migrate-env.sh

```
@@ -0,0 +1,229 @@
+# ALL app config (secrets, room IDs, feature flags) belongs in
+# /opt/bisq-support/docker/.env — the single source of truth for
+# Docker Compose.
+
+DEPLOY_HEADER
+
+    while IFS= read -r line || [ -n "$line" ]; do
+        stripped="${line#"${line%%[![:space:]]*}"}"
+        stripped="${stripped#export }"
+        [[ -z "$stripped" || "$stripped" == \#* ]] && continue
+
+        local var_name="${stripped%%=*}"
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)